lib/l10n/app_de.arb

```diff
@@ -1,6 +1,12 @@
 {
     "@@locale": "de",
 
+    "price": "Preis",
+    "cancel": "abbrechen",
+    "confirm": "bestätigen",  
+    "seat": "Platz",
```

Comment:
this should be done with pluralization. Something like (untested)

 "seat" : "{seats, plural, zero{Plätze} one{Platz} other{Plätze}}",
"@seat": {
  "placeholders": {
      "seats": {}
    }
}

Reply:
maybe the "zero" thing is unnecessary because it is included in "other" if not supplied, but I don't know

Reply:
yes i think zero is unnecessary, also because there can never be zero seats.